features/steps/promotion_steps.py

```diff
@@ -138,10 +125,6 @@ def step_impl(context, element_name):
     element = context.driver.find_element_by_id(element_id)
     expect(element.get_attribute('value')).to_be(u'')
 
-
-##################################################################
-# These two function simulate copy and paste
-##################################################################
```

Comment:
I'm thinking maybe we could leave these comments? So that whoever needs to work on these could understand the code easier?

Reply:
@liondon Hey Ling-I, I add the `active` filed in DB model cause according to create function in`rest_api.js`, we can set the `active` checkbox, that means it would generate an `active` data when creating a promotion, which needs a place to store in the database. 
And about change files in tests, since we add a new field(`active`) in DB model, there are some functions, like serialize, deserialize functions and so on, related to the `active` filed, so I think we need change these relevant files to make the tests work.

Reply:
@liondon I think if we do not want to include  the `active` filed in DB model, we could cancel the feature of setting `active` checkbox in create function( and could also cancel this in update function I think), and only leave the feature of setting `active` checkbox in search function.

Reply:
@Kristin01 yeah, I think the checkbox is just there for search, maybe we could just ignore the checkbox when we create promo in `rest_pi.js` script? @VickySteeves What do you think?

Reply:
@liondon Yeah, and I think we could also ignore`update_form_data` in `rest_pi.js`.

Reply:
@liondon @VickySteeves If we agree on the above changes ( to cancel  the feature of setting active checkbox (right now it changes to dropdown) of create, update promotion and `update_form_data` in `rest_pi.js`), I will make a modification now. Is that Okay?